bindings/python/src/OpenSpaceToolkitAstrodynamicsPy/Flight/Profile.cpp

```diff
@@ -55,7 +55,9 @@ inline void OpenSpaceToolkitAstrodynamicsPy_Flight_Profile(pybind11::module& aMo
 
         .value("GeocentricNadir", Profile::TargetType::GeocentricNadir, "Geocentric nadir")
         .value("GeodeticNadir", Profile::TargetType::GeodeticNadir, "Geodetic nadir")
-        .value("Trajectory", Profile::TargetType::Trajectory, "Trajectory")
+        .value("Trajectory", Profile::TargetType::Trajectory, "Trajectory")  // Deprecated in favor of TargetPosition
```

Comment:
nit: if there's a way to indicate a depracation in the bindings that would be great, if not no worries

Reply:
There isn't without fully overriding the bindings which I feel is a bit excessive.